### ISSUE TITLE
VIRUS

### ISSUE THREAD

**Issue by FPS-86** [laachouch8/PUBG-Hack-Ekzo-cheat-FREE-PUBG-cheat-Aim-ESP#1]
Don't download this shit! Trojan virus!!!!!!!!!!!!!!

**Comment by TMDhack** [comment]
> 


Is this thing useful to open? I open the egg, is it useful?

**Comment by FPS-86** [comment]
> 

If you want to get a Trojan, then open it.

**Comment by TMDhack** [comment]
> > 
> 
> 如果你想得到一个木马，然后打开它。

No, there is no response after I open the software and enter the game

**Comment by FPS-86** [comment]
> > > 
> > 
> > 
> > 如果你想得到一个木马，然后打开它。
> 
> No, there is no response after I open the software and enter the game

It will not be there, so a mining Trojan is sewn into the program, then you will see how your video card will heat up and how the PC lags

**Comment by TMDhack** [comment]
> > > > 
> > > 
> > > 
> > > 如果你想得到一个木马，然后打开它。
> > 
> > 
> > No, there is no response after I open the software and enter the game
> 
> It will not be there, so a mining Trojan is sewn into the program, then you will see how your video card will heat up and how the PC lags

Is there any cheating device that can be used on this platform? Give me a link

**Comment by FPS-86** [comment]
https://ivsofte.biz/pybg-hacks.html

**Comment by TMDhack** [comment]
> https://ivsofte.biz/pybg-hacks.html

thanks
